source/_integrations/rachio.markdown

```diff
@@ -78,6 +78,19 @@ The `rachio` switch platform allows you to toggle zones and schedules connected
 
 Once configured, a switch will be added for every zone that is enabled on every controller in the account provided and a switch to start or stop every schedule on a controller. There will also be a switch to toggle each controller's standby mode, as well as to activate a 24 hour rain delay on the device.
 
+## Service
+
+Rachio has one service that allows for setting the moisture percentage of a zone or group of zones. As Rachio only uses moisture levels for zones in a Flex Daily schedule, this service is only available when at least one zone is part of a Flex Daily schedule. 
+
+### Service `rachio.set_zone_moisture_percent`
+
+Set the zone moisture percentage for a zone or group of zones.
+
+| Service data attribute | Optional | Description |
+| ---------------------- | -------- | ----------- |
+| `entity_id` | no | String, list or group of zones to set moisture percentage.
```

Comment:
I guess I’m not following..Only devices, not entities, can be added to areas, correct? Adding a Rachio device to an area would add a bunch of other switches/sensors that wouldn’t be appropriate for this service. Sorry for the confusion.

Reply:
This service is unique for the rachio domain, if an area is targeted using this service, only entities provided by rachio will be triggered, others are ignored.

Reply:
Rachio uses switches for several different things like zones, schedules etc. The service only works with the zone portion. It seems like it would have errors with the other switch elements. I could be confused still.

Reply:
Sorry to hear that, but this service is registered using `async_register_entity_service`, which means `entity_id` is optional. Please adjust that.

See: <https://github.com/home-assistant/core/blob/a9ffc149f8abcd739b95a1f798e182f439a0cb2a/homeassistant/helpers/config_validation.py#L836>

Reply:
👍